pkg/collector/python/loader.go

```diff
@@ -38,7 +38,7 @@ import "C"
 var (
 	pyLoaderStats    *expvar.Map
 	configureErrors  map[string][]string
-	py3Warnings      map[string]string
+	py3Warnings      map[string][]string
```

Comment:
@hush-hush It seems `py3Warnings` wasn't used. I changed the type, let me know if that's an issue.

Reply:
shouldn't be a problem as far as I can tell, doesn't seem to be used before your PR indeed.